connectors/jira/src/test/java/com/vmware/connectors/jira/JiraControllerTest.java

```diff
@@ -72,26 +56,26 @@
 
     private MockRestServiceServer mockJira;
 
-    @Before
+    @BeforeEach
     public void setup() throws Exception {
```

Comment:
Can we remove ```public``` access specifier here as well?

Reply:
Please remove it in all the places.

Reply:
Done.